res/controllers/Numark-Scratch-scripts.js

```diff
@@ -0,0 +1,423 @@
+// eslint-disable-next-line no-var
+var NumarkScratch = {};
+
+/*
+ * USER CONFIGURABLE SETTINGS.
+ */
+
+// Defines the Beatloop Roll sizes for the 4 pads, for available values see:
+// https://manual.mixxx.org/2.4/en/chapters/appendix/mixxx_controls.html#control-[ChannelN]-beatlooproll_X_activate
+// Default: [0.25, 0.5, 1, 2]
+NumarkScratch.autoLoopSizes = [
+    "0.25",
+    "0.5",
+    "1",
+    "2",
+];
+
+// Defines how the Loop Encoder functions.
+// If 'true' the Encoder scrolls the library/loads track. Shift + Encoder manages looping.
+// If 'false' the Encoder manages looping. Shift + Encoder scrolls the library/loads track (Serato default).
+// Default: false
+NumarkScratch.invertLoopEncoderFunction = false;
+
+// Defines the bightness of button LEDs when they inactive.
+// '0x00' sets the LEDSs to completely off. '0x01 sets the LEDs to dim.
+// Default: 0x01 (dim)
+NumarkScratch.LOW_LIGHT = 0x01;
+
+/*
+ * CODE
+ */
+
+// State variable, don't touch
+NumarkScratch.shifted = false;
```

Comment:
I don't think this is necessary anymore.
```suggestion
```

Reply:
done